ISISComputingGroup/BeckhoffPLCCode
# Ticket 3987: Build Beckhoff Code

See https://github.com/ISISComputingGroup/IBEX/issues/3503

To test:
* Push some PLC code to this branch that won't build, confirm that it fails in Jenkins 
* Push good code and confirm it builds ok

Documentation is at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Beckhoff

This should be pair reviewed by @Simon-Cooper and an IBEX developer
